esmvaltool/config-references.yml

```
@@ -685,8 +685,9 @@ references:
   cru: "Harris et al., Int. J. Climatol., doi:10.1002/joc.3711, 2014."
   duveiller2018: "Duveiller et al., Sci. Data, doi:10.1038/sdata.2018.14, 2018."
   eppley-vgpm-modis: "Behrenfeld and Falkowski, Limnol. Oceanogr., doi:10.4319/lo.1997.42.1.0001, 1997."
-  era5: "Copernicus Climate Change Service (C3S) (2017): ERA5: Fifth generation of ECMWF atmospheric reanalyses of the global climate. Copernicus Climate Change Service Climate Data Store (CDS)"
   era-interim: "Dee et al., Q. J. Roy. Meteor. Soc., doi:10.1002/qj.828, 2011."
+  era-interim-land: "Balsamo et al. ERA-Interim/Land: a global land surface reanalysis data set, Hydrol. Earth Syst. Sci., 19, 389–407, https://doi.org/10.5194/hess-19-389-2015, 2015."
```

```suggestion
  era-interim-land: "Balsamo et al., Hydrol. Earth Syst. Sci., doi:10.5194/hess-19-389-2015, 2015."
```
To keep the formatting consistent.